charts/portworx-essentials/v1.0.1/values.yaml

```diff
@@ -0,0 +1,151 @@
+# Please uncomment and specify values for these options as per your requirements.
+kvdb:
+ownEtcdOption: none
+etcdAuth: none
+etcdType: "Built-in"                        # KVDB type
+
+etcd:
+  credentials: none:none              # Username and password for ETCD authentication in the form user:password
+  ca: none                            # Name of CA file for ETCD authentication. server.ca
+  cert: none                          # Name of certificate for ETCD authentication. Should be server.crt
+  key: none                           # Name of certificate key for ETCD authentication Should be server.key
+consul:
+  token: none                         # ACL token value used for Consul authentication. (example: 398073a8-5091-4d9c-871a-bbbeb030d1f6)
+region: none                          # US or EU regions for Portworx hosted etcds
+
+dataInterface: none                   # Name of the interface <ethX>
+managementInterface: none             # Name of the interface <ethX>
+platformOptions: none                 # AKS, EKS or GKE platforms
+
+customRegistryURL:
+registrySecret:
+
+clusterName: mycluster                # This is the default. please change it to your cluster name.
+secretType: k8s                      # Defaults to None, but can be AWS / KVDB / Vault.
+envVars: none                         # NOTE: This is a ";" seperated list of environment variables. For eg: MYENV1=myvalue1;MYENV2=myvalue2
+stork: true                           # Use Stork https://docs.portworx.com/scheduler/kubernetes/stork.html for hyperconvergence.
+storkVersion: 2.4.2
+
+lighthouse: false
+
+deployOnMaster: false                 # For POC only
+csi: false                            # Enable CSI
+
+serviceAccount:
+  hook:
+    create: true
+    name:
+
+deploymentType: oci                   # accepts "oci" or "docker"
+imageType: none                       #
+imageVersion: 2.5.5                 # Version of the PX Image.
```

Comment:
@sharma-tapas can we use a version that's released here?

Reply:
Since we have release 2.5.5. We can keep it as is.